tests/unit/main_html5/underflow-tests.js

```diff
@@ -457,6 +458,20 @@ describe('main_html5 chrome underflow tests', function () {
     expect(_.contains(vtc.notified, vtc.interface.EVENTS.WAITING)).to.be(true);
   });
 
+  it('should raise buffered event after waiting on MS Edge when currentTime has progressed', function(){
+    OO.isChrome = false;
```

Comment:
I recommend to set all OO.is{Browser,Platform} constants to false in afterEach - this way we won't depend on previous test to set to false constants that were used there